### ISSUE TITLE
Mirador crashes on Safari -- indexdb error

### ISSUE THREAD

**Issue by gigamorph** [yale-web-technologies/mirador-project#155]
```
Unhandled rejection: UnknownError: An unknown error occurred within Indexed Database.
H@http://mirador.lotb.yale.edu/assets/application-0871f7ef67bf551568912ef5cae38e6fe20e063b86c5ce02febcf64a47a518af.js:17:31750
B@http://mirador.lotb.yale.edu/assets/application-0871f7ef67bf551568912ef5cae38e6fe20e063b86c5ce02febcf64a47a518af.js:18:993
B@[native code]
http://mirador.lotb.yale.edu/assets/application-0871f7ef67bf551568912ef5cae38e6fe20e063b86c5ce02febcf64a47a518af.js:19:6461
```

**Comment by gigamorph** [comment]
Possible clue: https://bugs.webkit.org/show_bug.cgi?id=170000

**Comment by gigamorph** [comment]
Fixed.
Safari's version of indexeddb doesn't seem to be stable. I determined it is not worthwhile (for now) trying to find the root cause and/or waiting for the Safari code to be fixed. Instead I created a higher level cache (smaller max size, in-memory, JavaScript) so it can serve the immediate caching need. Indexeddb will still be used if it works, but if an operation fails as in Safari it will be marked invalid and not used from that point on.
